### ISSUE TITLE
Review comments in generated WinUI code

### ISSUE THREAD

**Issue by sibille** [microsoft/WindowsTemplateStudio#3996]
We should review the comments in the generated code to make sure all important links are mentioned and up to date.
We also saw there are a lot of instructions on removal of SampleData and think we should mention this only once as a TODO to not overload the tasklist.

Applies to the following platforms:
| UWP              | WPF              | WinUI           |
| :--------------- | :--------------- | :---------------|
| No | No |Yes |

**Comment by mvegaca** [comment]
#4027 Proposal for reduce TODO comments in WinUI 3 generated apps.

**Comment by mrlacey** [comment]
> #4027 Proposal for reduce TODO comments in WinUI 3 generated apps.

- What do you mean by calling this a proposal?
- If you want to "propose" making such changes why only relate this to the WinUI3 templates?

**Comment by sibille** [comment]
Goal of this issue was reviewing the comments on the WinUI templates. 
I think there are a bunch of other comments that we should include (see comment on the PR). 
I'll open a separate issue for reducing sample data related comments on UWP and WPF

**Comment by sibille** [comment]
Verified in dev-nightly version:
Templates version: 0.22.21022.1
Wizard version: 0.22.21022.1
